### ISSUE TITLE
Android Emulator Window is shaking and blinking when used in tiled mode

### ISSUE THREAD

**Issue by chpasha** [esjeon/krohnkite#72]
**Symptom**
Android Emulator window is shaking and blinking all the time until krohnkite is deactivated completely. 

**How to Reproduce**
Start android emulator from android sdk on krohnkite enabled screen

**Environment**
 - Distro: Debian 10
 - KWin version: 5.14.5
 - Krohnkite version: 0.7
 - List of KWin scripts in use: [Enforce Window Decoration on GTK]

**Notes**
Maybe the small floating tool window that appears outside of the main window could be the cause, perhaps it somehow makes krohnkite believe that the window geometry has changed. Unfortunately it doesn't seem to have an own window class or title, only window id, so I could not make krohnkite ignore it

**Comment by esjeon** [comment]
IIRC, other tiling WMs have problems with the emulator, too. The cause of the problem is the emulator, which uses an anti-pattern - periodic resize - to keep the window size fixed. The toolbar next to it has nothing to do with this.

A quick fix is to simply make the window *float*(<kbd>Win</kbd> + <kbd>F</kbd>). You can go further and ignore the window w/ title ("Android Emulator"), but this also has its own problems.

**Comment by chpasha** [comment]
ok, good to know. Unfortunately making it float defeats the purpose since I wanted it to share screen space with log window. Thank you anyway for quick response.

**Comment by esjeon** [comment]
> share screen space with log window

Yeah, that's one good thing about tiling WM. I'll see if I can safely detect repeated resizing attempts and prevent infinite wiggling.

**Comment by JohnDanielScanlon** [comment]
I've run into the same issue with krohnkite, using anbox and teamviewer, due to it trying to resize the window rapidly.

**Comment by skyf0cker** [comment]
> IIRC, other tiling WMs have problems with the emulator, too. The cause of the problem is the emulator, which uses an anti-pattern - periodic resize - to keep the window size fixed. The toolbar next to it has nothing to do with this.
> 
> A quick fix is to simply make the window _float_(Win + F). You can go further and ignore the window w/ title ("Android Emulator"), but this also has its own problems.

Really helpful, thanks
